src/components/Table/docs/Table.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import { LiveTable } from './LiveTable';
+import { ActiveRowTable } from './ActiveRowTable';
+import { ComplexDataTable } from './ComplexDataTable';
+import { SkeletonTable } from './SkeletonTable';
+import { TablePropsTable, TableRowPropsTable, TableRowSkeletonPropsTable } from './TablePropsTable';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks';
+
+# Table
+
+<Meta title="Components/Table" component={Table} />
```

Comment:
I think it would be good idea to aim to cover all props in argTypes with this migration (rowStyle, rowSize, columnSpace) with relevant controls per each prop type, so that our components are fully interactive on the Canvas tab (especially as this becomes a single view in Storybook v7, while Docs tab will be moved to sidebar)

This would apply to all migrated components :)